lib/Channel.js

```diff
@@ -215,7 +215,10 @@ class Channel {
     }
 
     sendMessage(body) {
-        return TwilioIPMessagingMessages.sendMessage(this.sid, body)
+        if(Platform.OS == 'android'){
+            this.initialize()
```

Comment:
I think `initialize()` returns a promise... should we wait to call send until we are in the `then()`?